schema/project-level/project-schema.json

```
@@ -414,6 +414,15 @@
             "$ref": "#/definitions/Modification"
           },
           "uniqueItems": true
+        },
+        "transactions": {
+          "title": "Transactions",
+          "description": "A list of the spending transactions made against this contracting process.",
```

I don't know if we have it in the style guide yet, but it's preferred not to describe the data type of the field.

```suggestion
          "description": "The spending transactions made against this contracting process.",
```